blog-comments-simple/Makefile

```diff
@@ -0,0 +1,10 @@
+
+readme:
```

Comment:
We should probably remove this, no? Considering it's going into a repo where we will just maintain the README.